nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/pom.xml

```diff
@@ -74,7 +74,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-core</artifactId>
-            <version>${jackson.version}</version>
+            <version>2.10.1</version>
```

Comment:
Is this change required? If yes, can we use the latest (2.10.3)?
Ideally, I would try to remain consistent but I agree that changing the Jackson version in the root pom can be quite impacting...

Reply:
Yes, it's required to use the latest sdk release. 2.10.1 matches the azure core sdk. I would stick to the azure version as compared to the latest. Let me know if it makes sense.

Reply:
@pvillard31 could you let me know if this PR is good to be merged or you have additional comments?

Reply:
No, it's good on my side. If @turcsanyip is a +1, happy to merge.